tests/test_in_memory.rs

```diff
@@ -550,14 +550,14 @@ async fn test_first_names() -> Result<(), Error<ExampleError>> {
     );
     println!("Built a list of {} search keywords", searches.len());
     println!(
-        "Graphs: table sizes: entry -> {} records, {} kbytes, chain -> {} records, {} kbytes",
+        "Graphs: table sizes: entry -> {} records, {} kb, chain -> {} records, {} kb",
```

Comment:
`kb` stands for kilo bits.

```suggestion
        "Graphs: table sizes: entry -> {} records, {} kB, chain -> {} records, {} kB",
```

Reply:
indeed!